src/composables/queries/useVotingEscrowQuery.ts

```diff
@@ -0,0 +1,75 @@
+import QUERY_KEYS from '@/constants/queryKeys';
+import useGraphQuery from './useGraphQuery';
+import useWeb3 from '@/services/web3/useWeb3';
+import { Network } from '@/lib/config';
+
+export const networkSubgraphGagesMap = {
+  [Network.MAINNET]:
+    'https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-gauges',
+  [Network.ARBITRUM]:
+    'https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-gauges-arbitrum',
+  [Network.POLYGON]:
+    'https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-gauges-polygon',
+  [Network.OPTIMISM]:
+    'https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-gauges-optimism',
+  [Network.GNOSIS]:
+    'https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-gauges-gnosis-chain',
+};
```

Comment:
These gauge subgraph URLs are already defined in configs, please pull them from there.